### PR TITLE
Refactored e2e test - fixed python for no wgs submission - layout fixes in SNP map show table button - Date fixes

### PR DESCRIPTION
**Pull request by praveenmurugan101**
- Ignore test cases submission id to avoid getting capitalised in ValidateIdentifierInput function
- Date fixes for non bovine submission
-  Refactored e2e test codes for the latest UI updates

### CODE REVIEW

File: app/static/js/main.js [hunk 1]
```diff
@@ -309,6 +309,11 @@ async function loadStaticContent(tab){
 
  window.validateIdentifierInput = function(identifierStr){
 // regex to remove all spaces and method to make all alphabets uppercase
+
+//exclude test cases
```
[the remark this answers is not in the document]
Comment (ellisrichardj) [reply app/static/js/main.js]: Isn't this about making what the user inputs case-insensitive?  Just trying a few other options out

[the remark this answers is not in the document]
Comment (praveenmurugan101) [reply app/static/js/main.js]: We are trying to include case insensitive query in SQL search, it is one of the options

**Comment by televisedbrain** [comment]
mate I have no idea why changing the sql statement broke everything

**Review by ellisrichardj — APPROVED** [review]
Great that all the tests are now passing so happy to approve 👍

**Review by OmarSainab — APPROVED** [review]
Changes look good 👍  maybe update the PR comment as changes in main.js are no longer ignoring test cases

**Pull request merged**